citizen/citizen.c

```diff
@@ -184,30 +194,38 @@ void printCitizen(Citizen citizen) {
 }
 
 // remove a citizen from the list based on citizenId.
-ResponseCode removeCitizen(List *list, int citizenId) {
+ResponseCode removeCitizen(List *list, int citizenId) 
```

Comment:
checked

Reply:
please follow the naming convention on the variables, use snake_case

Reply:
Done. Thanks.